api/src/main/java/org/openmrs/module/fhir2/providers/util/FhirProviderUtils.java

```diff
@@ -33,6 +36,22 @@ public static MethodOutcome buildCreate(DomainResource resource) {
 		return buildWithResource(methodOutcome, resource);
 	}
 	
+	public static OperationOutcome buildDelete(OperationOutcome.IssueSeverity code, OperationOutcome.IssueType issueType,
+	        BaseServerResponseException exception) {
+		//created codeable concepts to be called by operationOutcome, codeableconcept only limited to two properties text and coding
+		CodeableConcept codeableConcept = new CodeableConcept().setText("MSG_DELETED").setText("informational")
+		        .setText("This resource has been deleted").setText("informational")
+		        .setText("http://terminology.hl7.org/CodeSystem/operation-outcome").setText("This resource has been deleted")
+		        .setText(exception.getMessage());
+		
+		OperationOutcome operationOutcome = new OperationOutcome();
+		operationOutcome.addIssue().setSeverity(OperationOutcome.IssueSeverity.ERROR).setCode(issueType)
```

Comment:
i set issueSeverity.ERROR  as it should be supposed to be

Reply:
@sherrif10 Actually, this should be `IssueSeverity.INFORMATION, because we're sending a result that says the object was successfully deleted. No error occurred. I've made that change.

Reply:
Thanks for replying @ibacher , Wont we need to call these errors as variables, i mean something like publi static string INFORMATION="A message that the server will send";. this is my idea however, or we just define the extension of issueSeverity and the server knows the what to send!